app/uk/gov/hmrc/transitmovementsrouter/models/MovementType.scala

```diff
@@ -16,4 +16,11 @@
 
 package uk.gov.hmrc.transitmovementsrouter.models
 
+import play.api.libs.json.Format
+import play.api.libs.json.Json
+
 case class MovementType(value: String) extends AnyVal
```

Comment:
2 movement objects:   Departure and Arrival?

Reply:
again, it's an existing class - not sure will add any value now for this as we end up changing loads of files if we change this.